src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
```

Comment:
normal is default value, you can remove this style

Reply:
Done